src/view/abstract.ts

```diff
@@ -1,9 +1,9 @@
-import { AnyObj, Elem, State } from '../types.js'
+import { AnyObj, Elem, State } from '../constants.js'
 import ViewStore from '../store/viewStore.js'
 import el from '../util/dom.js'
 import errorHandler from '../util/errorHandler.js'
 
-const eventErrorCatcher = (handler: any) => (e: CustomEvent) => {
+const eventHandlerWithValidation = (handler: any) => (e: CustomEvent) => {
```

Comment:
```suggestion
const eventHandlerWithValidation = (handler: Function) => (e: CustomEvent) => {
```
핸들러니까 any보다는 함수 타입을 지정해도 괜찮을꺼 같습니다

Reply:
네, step3에서 가능한 any를 모두 제거해보도록 하겠습니다.